tests/setup.js

```diff
@@ -75,6 +75,23 @@ afterEach(() => {
     }
 });
 
+jest.mock('react-intl', () => {
```

Comment:
Why do we need to partly mock this? The hook isn't supported by Jest?

Reply:
Did this while testing `OverlayTrigger`, no longer needed; should've removed. (Done now)